nestkernel/connection_manager.h

```diff
@@ -294,6 +295,10 @@ class ConnectionManager : public ManagerInterface
    */
   DelayChecker& get_delay_checker();
 
+  // public stop watch for benchmarking purposes
+  // start and stop in high-level connect functions in nestmodule.cpp and nest.cpp
+  Stopwatch sw_construction_connect;
```

Comment:
This should have a doxygen documentation comment, e.g. `Stopwatch measuring connection time`.